qlib/utils/paral.py

```
@@ -88,32 +88,28 @@ def __init__(self) -> None:
         self._t.start()
 
     def close(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)